version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.1.12-SNAPSHOT"
+version in ThisBuild := "0.1.14-SNAPSHOT"
```

Comment:
Shouldn't this be changed only by the release process?

Reply:
I ran `sbt release` on the feature branch since the changes are small and this library is only used by Identity API